pkg/cmd/server/origin/ensure.go

```diff
@@ -32,7 +32,10 @@ func (c *MasterConfig) ensureOpenShiftSharedResourcesNamespace() {
 		_, err = c.KubeClient().Namespaces().Create(namespace)
 		if err != nil {
 			glog.Errorf("Error creating namespace: %v due to %v\n", namespace, err)
+			return
 		}
+
+		c.ensureNamespaceServiceAccountRoleBindings(c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace)
```

Comment:
three references... save `c.Options.PolicyConfig.OpenShiftSharedResourcesNamespace` to a local var?